### ISSUE TITLE
Nice Job!

### ISSUE THREAD

**Issue by briangx92** [RamshackleJohnny/more-css-practice#2]
I like what you did with your Amazon page by creating the tables, that's really neat. I think the professor said that we're not allowed to use the ``<br>`` code though

**Comment by zachfedor** [comment]
@briangx92 Good call, we should avoid using `<br>` elements if we are using them for styling purposes instead of using CSS. In this case, @RamshackleJohnny could put `display: block;` on the images and the text would be put on its own line.

Also, we shouldn't be using tables for styling either. Tables are for grids of data, not for manufacturing a layout. We learned CSS positioning for a reason  ;)
